src/components/WebexAvatar/WebexAvatar.scss

```diff
@@ -32,16 +51,24 @@
     }
   }
 
-  .avatar-icon-active {
-    color: var(--wxc-person-status--active-color);
-  }
-  .avatar-icon-in-a-meeting {
-    color: var(--wxc-person-status--in-a-meeting-color);
-  }
-  .avatar-icon-dnd {
-    color: var(--wxc-person-status--dnd-color);
-  }
-  .avatar-icon-away {
-    color: var(--wxc-person-status--away-color);
-  }
+  .status-active { color: var(--wxc-person-status--active-color); }
+  .status-call { color: var(--wxc-person-status--in-a-meeting-color); }
+  .status-meeting { color: var(--wxc-person-status--in-a-meeting-color); }
+  .status-dnd { color: var(--wxc-person-status--dnd-color); }
+  .status-presenting { color: var(--wxc-person-status--dnd-color); }
+  .status-inactive { color: var(--wxc-person-status--away-color); }
+  .status-ooo { color: var(--wxc-person-status--away-color); }
+
 }
+
+.avatar-color-default { background-color: var(--wxc-avatar--color-default); }
```

Comment:
I would use
```css
.placeholder-1 { background: var(--wxc-avatar--placeholder-1--background); }
.placehodler-2 { background: var(--wxc-avatar--placeholder-2--background); }
```
this way, the placeholder can be further customized (eg: different text color, border) without changing the semantics.